conf/deployment/fusion_hci_pc/client_bm_upi_1az_rhcos_nvme_3m_3w.yaml

```diff
@@ -0,0 +1,13 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetal'
+  cluster_type: 'hci_client'
+  cluster_namespace: "openshift-storage-client"
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
```

Comment:
Do we indeed have 3 masters? I remember it was mentioned that the node types differ on Fusion HCI from regular 3+3 deployment